### ISSUE TITLE
Please add site https://Ranobes.TOP/

### ISSUE THREAD

**Issue by Arsur3215Loki** [dteviot/WebToEpub#946]
Ranobes is switching addresses, I don't know if what was used in ranobes.net works in ranobes.top.

**Comment by dteviot** [comment]
@Arsur3215Loki 

> I don't know if what was used in ranobes.net works in ranobes.top.

Works for https://ranobes.top/novels/943277-blood-warlock-v741610.html

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.

For my notes: 21 minutes work

**Comment by Arsur3215Loki** [comment]
> @Arsur3215Loki
> 
> > I don't know if what was used in ranobes.net works in ranobes.top.
> 
> Works for https://ranobes.top/novels/943277-blood-warlock-v741610.html
> 
> Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
> 
> For my notes: 21 minutes work

I got it to work using the Test Version, Thank you.

**Comment by DicerX** [comment]
> @Arsur3215Loki
> 
> > I don't know if what was used in ranobes.net works in ranobes.top.
> 
> Works for https://ranobes.top/novels/943277-blood-warlock-v741610.html
> 
> Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
> 
> For my notes: 21 minutes work

Yup this works like a charm. Any idea why the official extension still hasn't been updated.

**Comment by dteviot** [comment]
@Arsur3215Loki, @DicerX

I had RL issue for 6 weeks (Father in hospital with critical health issue).  Then I developed an addition to https://www.satisfactorygame.com/.  Actually, I still have an addition, which is why progress is currently nearly stopped.

Anyway, Updated version (0.0.0.154) has been submitted to Firefox and Chrome stores.
Firefox version is available now.
Chrome might be available in 1 to 3 weeks.

Updated version has fix for Ranobes URL change.
